src/controllers/numbersControllers.js

```diff
@@ -0,0 +1,89 @@
+const { add, subtract, multiply, divide, remainder } = require('./../lib/numbers');
+
+exports.add = (req, res) => {
+  const a = parseInt(req.params.a);
+  const b = parseInt(req.params.b);
+
+  return Number.isNaN(a) || Number.isNaN(b)
+    ? res.status(400).json({
+        error: 'Parameters must be valid numbers.',
+      })
+    : res.status(200).json({
+        result: add(a, b),
+      });
+};
+
+exports.subtract = (req, res) => {
+  const a = parseInt(req.params.a);
+  const b = parseInt(req.params.b);
+
+  return Number.isNaN(a) || Number.isNaN(b)
+    ? res.status(400).json({
+        error: 'Parameters must be valid numbers.',
```

Comment:
Usually when you do a ternary or some if else logic, you tend to put the most likely or frequently occurring option first. So if you expect your function to be used correctly most of the time (also known as the happy path), we would put the 200 success status first